<launch>
    <!-- arguments -->
    <!-- mb different for different experiments -->
    <rosparam param="mysql_data_saver_experiment_description">
        {
        experiment_type: 'test',
        machine_id: 'farmer4',
        experiment_legend: 'test table-search experiment with IRGA',
        start_datetime: '2020_10_18 23:00:00',
        end_datetime: '2020_10_30 23:00:00',
        additional_notes: 'add any notes about experiment errors or smth other important',
        experiment_id: '2020_07_03_TESTTYPE_0001',
        experiment_number: '2'
        }
    </rosparam>


    <!-- mb different for different experiments -->
     <rosparam param="exp_search_table">
         [
    {"number": 1, "red": 130, "white": 130},
    {"number": 2, "red": 70, "white": 190},
    {"number": 3, "red": 190, "white": 70},
    {"number": 4, "red": 40, "white": 160},
    {"number": 5, "red": 160, "white": 40},
    {"number": 6, "red": 100, "white": 100},
    {"number": 7, "red": 220, "white": 220},
    {"number": 8, "red": 25, "white": 235},
    {"number": 9, "red": 145, "white": 115},
    {"number": 10, "red": 85, "white": 55},
    {"number": 11, "red": 205, "white": 175},
    {"number": 12, "red": 55, "white": 85},
    {"number": 13, "red": 175, "white": 205},
    {"number": 14, "red": 115, "white": 145},
    {"number": 15, "red": 235, "white": 25},
    {"number": 16, "red": 17, "white": 138}
    ]
    </rosparam>

    <arg name="control_start_mode" default="full_experiment"/>

    <include file="$(find ros_farmer_pc)/launch/control_sba_led_relay_serial_test.launch" >
            <arg name="arduino_baudrate" default="115200"/>
    </include>
    <include file="$(find ros_farmer_pc)/launch/mysql_data_saver_test.launch"/>

    <node pkg="ros_farmer_pc" type="exp_system.py" name="exp_system">

    </node>


</launch>